src/main/kotlin/nl/info/zac/productaanvraag/ProductaanvraagService.kt

```diff
@@ -70,7 +80,10 @@ class ProductaanvraagService @Inject constructor(
     private val zrcClientService: ZrcClientService,
     private val drcClientService: DrcClientService,
     private val ztcClientService: ZtcClientService,
+    private val klantClientService: KlantClientService,
```

Comment:
3 more dependencies from the ProductaanvraagService... :-( Is this really really needed? The ProductaanvraagService is already way too big, so our goal is to reduce it in size and dependencies. This story calls for more refactoring here I feel instead of just adding only more complexity to it..